packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -46,10 +46,7 @@
 							}}
 						</n8n-text>
 					</div>
-					<MainSidebarVersionControl
-						v-if="versionControlStore.preferences.connected"
-						:is-collapsed="isCollapsed"
-					/>
+					<MainSidebarVersionControl :is-collapsed="isCollapsed" />
```

Comment:
This doesn't seem to look right.. Can you double check? 

<img width="223" alt="image" src="https://github.com/n8n-io/n8n/assets/6179477/a155f6b1-367a-4e9b-91d5-b18e0cb03e31">

Reply:
Fixed